voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/alexa/FlashBriefingResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.derpgroup.derpwizard.voice.model.alexa;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+
+@SuppressFBWarnings
+public class FlashBriefingResponse {
+  
+  public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'.0Z'");
+  
+  private String uid;
+  private String titleText;
+  private String mainText;
+  @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd'T'HH:mm:ss'.0Z'", timezone="UTC")
+  private Date updateDate;
```

Comment:
Is it possible to use a better time-related class? I thought we were supposed to be moving away from Date. Something like ZonedDateTime:
http://stackoverflow.com/questions/31627992/spring-data-jpa-zoneddatetime-format-for-json-serialization